### DISCUSSION TITLE
Could the junrar project benefit from free security help?

### DISCUSSION THREAD

**Discussion by Amir-Montazery** [junrar/junrar#107] [Ideas]
Hello junrar community! Open Source Technology Improvement Fund is piloting out helping critical projects like junrar with their security needs. We have some resources dedicated to helping improve security posture and tooling. I wasn't sure how best to reach out. Please let me know if this sounds interesting and who to connect with. Thank you in advance!

**Comment by gotson** [comment]
Hi, could you maybe expand on what kind of help you would be willing to offer?

**Reply by Amir-Montazery** [reply]
Yes absolutely! We are offering a collaborative security and tooling review. One of our teams will work with you to understand junrar's security needs and how they can best help improve the security posture of the project. That can come in the form of manually reviewing parts of the code base, finding and fixing vulnerabilities, implementing fuzzing or other SAST tools, and general security improvements.

**Reply by gotson** [reply]
I don't have any time to spend, so i'm not sure if this is going to work tbh

**Reply by Amir-Montazery** [reply]
That is certainly understandable. No worries. Do you think any of the other junrar contributors/maintainers would be interested?

**Reply by gotson** [reply]
I am the only maintainer

**Reply by Amir-Montazery** [reply]
I understand. Thank you for taking the time to consider. If you need help with anything security-related in the future, let me know and I'd be happy to see how OSTIF can help! Thanks again :)
